subsys/bluetooth/controller/hci_internal.c

```diff
@@ -762,6 +762,14 @@ static uint8_t vs_cmd_put(uint8_t const * const cmd,
 #endif /* CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL */
 #endif /* CONFIG_BT_HCI_VS_EXT */
```

Comment:
`SDC_HCI_OPCODE_CMD_VS_ZEPHYR_READ_CHIP_TEMP` belongs to `CONFIG_BT_HCI_VS_EXT` (Zephyr HCI Vendor-Specific Extensions). Therefore, this line must be moved down.